android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -113,6 +113,7 @@
   // Instance state
   private static final String STATE_PP_OPENED = "PpOpened";
   private static final String STATE_MAP_OBJECT = "MapObject";
+  private static final String STATE_SHOW_TIME_LEFT = "ShowTimeLeft";
```

Comment:
When you pass the whole bundle state (see my notes below) to the mNavigationController class you will be able to move this constant to NavigationController class (for example, take a look at the `RoutingPlanInplaceController `)